Pipfileでのoletoolsの参照が404エラーになった

https://github.com/takeruko/extract_vba_source/blob/aff8889a84aa42a4b4379db0aee5203cfc4c7931/Pipfile#L7 を原因として `pipenv install` でエラーが発生するようになりました。

<details>
<summary>該当エラーログ</summary>

```
Pipfile.lock not found, creating...
Locking [packages] dependencies...
Building requirements...
CRITICAL:pipenv.patched.pip._internal.network.download:HTTP error 404 while getting https://github.com/kijeong/oletools/archive/refs/heads/add/projectcompatversion_record.zip#egg=oletools
Traceback (most recent call last):
  File "/opt/hostedtoolcache/Python/3.10.14/x64/bin/pipenv", line 8, in <module>
    sys.exit(cli())
  File "/opt/hostedtoolcache/Python/3.10.14/x64/lib/python3.10/site-packages/pipenv/vendor/click/core.py", line 1130, in __call__
    return self.main(*args, **kwargs)
  File "/opt/hostedtoolcache/Python/3.10.14/x64/lib/python3.10/site-packages/pipenv/cli/options.py", line 58, in main
    return super().main(*args, **kwargs, windows_expand_args=False)
  File "/opt/hostedtoolcache/Python/3.10.14/x64/lib/python3.10/site-packages/pipenv/vendor/click/core.py", line 1055, in main
    rv = self.invoke(ctx)
  File "/opt/hostedtoolcache/Python/3.10.14/x64/lib/python3.10/site-packages/pipenv/vendor/click/core.py", line 1657, in invoke
    return _process_result(sub_ctx.command.invoke(sub_ctx))
  File "/opt/hostedtoolcache/Python/3.10.14/x64/lib/python3.10/site-packages/pipenv/vendor/click/core.py", line 1404, in invoke
    return ctx.invoke(self.callback, **ctx.params)
  File "/opt/hostedtoolcache/Python/3.10.14/x64/lib/python3.10/site-packages/pipenv/vendor/click/core.py", line 760, in invoke
    return __callback(*args, **kwargs)
  File "/opt/hostedtoolcache/Python/3.10.14/x64/lib/python3.10/site-packages/pipenv/vendor/click/decorators.py", line 84, in new_func
    return ctx.invoke(f, obj, *args, **kwargs)
  File "/opt/hostedtoolcache/Python/3.10.14/x64/lib/python3.10/site-packages/pipenv/vendor/click/core.py", line 760, in invoke
    return __callback(*args, **kwargs)
  File "/opt/hostedtoolcache/Python/3.10.14/x64/lib/python3.10/site-packages/pipenv/cli/command.py", line 233, in install
    do_install(
  File "/opt/hostedtoolcache/Python/3.10.14/x64/lib/python3.10/site-packages/pipenv/routines/install.py", line 170, in do_install
    do_init(
  File "/opt/hostedtoolcache/Python/3.10.14/x64/lib/python3.10/site-packages/pipenv/routines/install.py", line 768, in do_init
    do_lock(
  File "/opt/hostedtoolcache/Python/3.10.14/x64/lib/python3.10/site-packages/pipenv/routines/lock.py", line 65, in do_lock
    venv_resolve_deps(
  File "/opt/hostedtoolcache/Python/3.10.14/x64/lib/python3.10/site-packages/pipenv/utils/resolver.py", line 1057, in venv_resolve_deps
    deps = convert_deps_to_pip(deps, project, include_index=True)
  File "/opt/hostedtoolcache/Python/3.10.14/x64/lib/python3.10/site-packages/pipenv/utils/dependencies.py", line 279, in convert_deps_to_pip
    req = new_dep.as_line(
  File "/opt/hostedtoolcache/Python/3.10.14/x64/lib/python3.10/site-packages/pipenv/vendor/requirementslib/models/requirements.py", line 2587, in as_line
    assert self.line_instance is not None
  File "/opt/hostedtoolcache/Python/3.10.14/x64/lib/python3.10/site-packages/pipenv/vendor/requirementslib/models/requirements.py", line 2399, in line_instance
    return self.get_line_instance()
  File "/opt/hostedtoolcache/Python/3.10.14/x64/lib/python3.10/site-packages/pipenv/vendor/requirementslib/models/requirements.py", line 2363, in get_line_instance
    version, fetched = self.get_version_from_setup_info()
  File "/opt/hostedtoolcache/Python/3.10.14/x64/lib/python3.10/site-packages/pipenv/vendor/requirementslib/models/requirements.py", line 2750, in get_version_from_setup_info
    self.req.parse_setup_info()
  File "/opt/hostedtoolcache/Python/3.10.14/x64/lib/python3.10/site-packages/pipenv/vendor/requirementslib/models/requirements.py", line 1571, in parse_setup_info
    self.setup_info = SetupInfo.from_ireq(
  File "/opt/hostedtoolcache/Python/3.10.14/x64/lib/python3.10/site-packages/pipenv/vendor/requirementslib/models/setup_info.py", line 1697, in from_ireq
    unpack_url(
  File "/opt/hostedtoolcache/Python/3.10.14/x64/lib/python3.10/site-packages/pipenv/vendor/requirementslib/models/setup_info.py", line 1837, in unpack_url
    file = get_http_url(
  File "/opt/hostedtoolcache/Python/3.10.14/x64/lib/python3.10/site-packages/pipenv/vendor/requirementslib/models/setup_info.py", line 1802, in get_http_url
    from_path, content_type = download(link, temp_dir.path)
  File "/opt/hostedtoolcache/Python/3.10.14/x64/lib/python3.10/site-packages/pipenv/patched/pip/_internal/network/download.py", line 134, in __call__
    resp = _http_get_download(self._session, link)
  File "/opt/hostedtoolcache/Python/3.10.14/x64/lib/python3.10/site-packages/pipenv/patched/pip/_internal/network/download.py", line 118, in _http_get_download
    raise_for_status(resp)
  File "/opt/hostedtoolcache/Python/3.10.14/x64/lib/python3.10/site-packages/pipenv/patched/pip/_internal/network/utils.py", line 54, in raise_for_status
    raise NetworkConnectionError(http_error_msg, response=resp)
pipenv.patched.pip._internal.exceptions.NetworkConnectionError: 404 Client Error: Not Found for url: https://codeload.github.com/kijeong/oletools/zip/refs/heads/add/projectcompatversion_record
Error: Process completed with exit code 1.
```
</details>

代替として以下のようにすることで問題を回避しています。
`oletools = {file = "https://github.com/kijeong/oletools/archive/refs/heads/master.zip"}`

--- a/Pipfile
+++ b/Pipfile
@@ -4,4 +4,4 @@ verify_ssl = true
 name = "pypi"
 
 [packages]
-oletools = {file = "https://github.com/kijeong/oletools/archive/refs/heads/add/projectcompatversion_record.zip"}
+oletools = {file = "https://github.com/kijeong/oletools/archive/refs/heads/master.zip"}

